docs/source/_topic_map.yml

```diff
@@ -25,6 +25,8 @@ Topics:
         File: managing-minishift
       - Name: Interacting with Openshift
         File: interacting-with-openshift
+      - Name: Add-ons
```

Comment:
This should go above "Interacting with OpenShift". In the order of things, it comes before interacting with OpenShift. That said, it should also be "OpenShift"